assets/js/src/core/modules/app/base-layout/user-menu/user-menu.tsx

```diff
@@ -9,24 +9,25 @@
  */
 
 import { Icon } from '@Pimcore/components/icon/icon'
-import { Avatar, type MenuProps } from 'antd'
-import React from 'react'
+import { Badge, Avatar, type MenuProps } from 'antd'
+import React, { useState } from 'react'
 import { Dropdown } from '@Pimcore/components/dropdown/dropdown'
 import { useTranslation } from 'react-i18next'
 import { useStyle } from './user-menu.styles'
 import { useLogoutMutation } from '@Pimcore/modules/auth/authorization-api-slice.gen'
 import trackError, { ApiError } from '@Pimcore/modules/app/error-handler'
 import { NOTIFICATIONS } from '@Pimcore/modules/notifications'
-import { Badge, Button } from '@sdk/components'
```

Comment:
Was something wrong with this import?

Reply:
we never used the sdk import. I replaced it with the ant import

Reply:
used pimcore import for badge and button https://github.com/pimcore/studio-ui-bundle/pull/1515/commits/f42ba057c01d41a8d98934df4ddc12e54b6aa657